### ISSUE TITLE
8.5 Permutation Feature Importance 

### ISSUE THREAD

**Issue by alexhallam** [christophM/interpretable-ml-book#351]
Great book. 

One comment I would add.

I think that another disadvantage of permutation feature importance is directionality. In some applications it is not enough to know how important a feature is, but if that feature moved the response up or down. To my knowledge permutation feature importance does not provide directionality.

**Comment by christophM** [comment]
Thanks for the feedback. I currently see feature importance as something for ranking features.

Directionality is something I consider that feature effects or feature attributions have.

**Comment by alexhallam** [comment]
I agree. I am just making the point that "directionality" is not listed as a disadvantage of permutation feature importance. Maybe it does not need to be. I just wanted to say that the book does not point this out explicitly.

**Comment by christophM** [comment]
Yeah, I see now what you mean. Maybe I'll add that in the 3rd edition of the book.
